src/main/java/CinemaRoomRestService/RestService/domain/entity/Statistics.java

```diff
@@ -0,0 +1,25 @@
+package CinemaRoomRestService.RestService.domain.entity;
+
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+@Getter
+public class Statistics {
+    private int currentIncome;
+    private int availableSeats;
+    private int purchasedTickets;
+
+    @Autowired
+    public Statistics(List<Seat> list) {
+        availableSeats = list.size();
+        for (Seat current : list) {
+            if (!current.isFree) {
+                availableSeats--;
```

Comment:
Do you really need to calculate that every time in a loop?? You know all your availiable seats. purchasedTickets = ALL_SEATS - availableSeats ?